snap/hooks/connect-plug-intel-npu-fw

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [ "${SNAP_UID}" -ne 0 ]; then
+  >&2 echo "Please run as root"
+  exit 1
+fi
+
+cd "${SNAP}"
+fw_blobs=$(find "${SNAP_DATA}" -name "*.bin")
+if [ -z "${fw_blobs}" ]; then
+  >&2 echo "No binary blobs found in ${SNAP_DATA}."
+  >&2 echo "Not updating firmware search path."
+  exit 1
+fi
+
+CUSTOM_FW_PATH_FILE=/sys/module/firmware_class/parameters/path
+echo -n "${SNAP_DATA}" > "${CUSTOM_FW_PATH_FILE}"
```

Comment:
since this is system wide configuration, is this better to append into that file ? or if this is complex to do at the moment and out of scope, drop a comment here or in the README ?

Reply:
After a closer look at the documentation, I think that it is a parameter of firmware_class module and it takes only 1 path (up to 256 characters), so that is fine